mandatory/3-function-output.js

```diff
@@ -5,12 +5,15 @@ function getRandomNumber() {
 
```

Comment:
can you explain what the first 2 functions do?

Reply:
I have a function **getRandomNumber()** without parameter inside the function there is another function The **Math. random() function** returns a floating-point, pseudo-random number in the range 0 to less than 1 (inclusive of 0, but not 1) with approximately uniform distribution over that range — which you can then scale to your desired range
// expected output: a number from 0 to <10